app/channels/application_cable/connection.rb

```diff
@@ -1,4 +1,20 @@
 module ApplicationCable
   class Connection < ActionCable::Connection::Base
+    identified_by :current_user
+
+    def connect
+      self.current_user = find_user
+    end
+
+    private
+    def find_user
+      if user = User.find_by(token: cookies.encrypted[:user_token])
```

Comment:
Alternative:
```rb
    User.where(token: cookies.encrypted[:user_token]).first_or_initialize do |user|
      user.token = User.generate_token
      user.ip = request.remote_ip
      user.save!
    end
```

Also, this could live in the User model